src/js/components/CheckBox/stories/Temp.js

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from 'react';
+
+import {
+  Box,
+  Grommet,
+  CheckBox,
+  FormField,
+  DataTable,
+  ThemeContext,
+} from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+import { hpe } from 'grommet-theme-hpe';
```

Comment:
Let's remove this import after the PR is approved, it's good for POC but not needed on a real deployment, we can use test-cafe for this type of testing.

Reply:
I think we should remove this story altogether as I don't see it adding value outside the context of this PR demonstrating the differences. I had just included it to help reviewers visualize the different behaviors.

> #### What testing has been done on this PR?
> Storybook using the story "input-checkbox--temp" which will be removed from the PR, but is useful for testing.
> 
> #### How should this be manually tested?
> Storybook - Checkbox/stories/Temp.js
>